public_html/wp-content/plugins/pattern-creator/src/components/unit-control/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+/* eslint-disable-next-line @wordpress/no-unsafe-wp-apis -- Experimental is OK. */
+import { __experimentalUnitControl as Control } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Modules constants
+ */
+const UNITS = [
+	{ value: 'px', label: 'px' },
+	{ value: '%', label: '%' },
+	{ value: 'em', label: 'em' },
+	{ value: 'rem', label: 'rem' },
+	{ value: 'vw', label: 'vw' },
+];
```

Comment:
You can address this in a follow-up PR/issue, but the only value supported by patterns is px, so we'll need to figure out conversions or tweak this control.

Reply:
Ha, dang. Getting all the units working was a big pain. :)